### PR TITLE
Clean up Docker files

### PR DESCRIPTION
**Pull request by egede**

### CODE REVIEW

**Review by alexanderrichards — DISMISSED** [review]

**Comment by egede** [comment]
We take the latest Dirac version to avoid that we get stuck on an old one and forget about it. Judged to be the minor of two evils

**Review by mesmith75 — APPROVED** [review]

File: ganga/GangaCore/Dockerfile [hunk 1]
```diff
@@ -1,20 +1,8 @@
-FROM centos:6
-LABEL maintainer "Alexander Richards <a.richards@imperial.ac.uk>"
-ARG ganga_version=6.5.2
+FROM centos:7
+LABEL maintainer "Ulrik Egede <ulrik.egede@monash.edu>"
+ARG ganga_version=8.4.2
```
Comment (mesmith75) [inline ganga/GangaCore/Dockerfile:3]: Would we rather have this as the latest release

File: .github/workflows/release_workflow.yml [hunk 2]
```diff
@@ -29,7 +29,8 @@ jobs:
         echo "::set-env name=VERSION::${VERSION}"
         sed --in-place "s/^_gangaVersion = .*/_gangaVersion = '${VERSION}'/g" ./setup.py
         sed --in-place "s/^_gangaVersion = .*/_gangaVersion = '${VERSION}'/g" ganga/GangaCore/__init__.py
-        git add ./setup.py ganga/GangaCore/__init__.py
+        sed --in-place "s/^ARG ganga_version=.*/ARG ganga_version='${VERSION}'/g" ganga/GangaCore/Dockerfile
```
Comment (egede) [inline .github/workflows/release_workflow.yml:32]: @mesmith75 This line is updating the version number in the Dockerfile to the latest release.

Comment (mesmith75) [reply .github/workflows/release_workflow.yml:32]: Ah yes, missed it.

**Pull request merged**